src/sentry/api/base.py

```diff
@@ -450,6 +454,20 @@ def dispatch(self, request: Request, *args, **kwargs) -> Response:
                     span.set_data("SENTRY_API_RESPONSE_DELAY", settings.SENTRY_API_RESPONSE_DELAY)
                     time.sleep(settings.SENTRY_API_RESPONSE_DELAY / 1000.0 - duration)
 
+        self.response = self.finalize_response(request, response, *args, **kwargs)
+
+        # Only enforced in dev environment
```

Comment:
is this comment correct? also enforced in tests, right?

Reply:
Hmm, I am not sure why its picking it up as a change I made, its in master as well. https://github.com/getsentry/sentry/blob/master/src/sentry/api/base.py#L440

Reply:
I don't see it in the file you linked?

Reply:
https://github.com/getsentry/sentry/blob/de0d46eb1ee2a80fc0283791ec77d1da772bf57c/src/sentry/api/base.py#L440

Reply:
https://github.com/getsentry/sentry/blob/de0d46eb1ee2a80fc0283791ec77d1da772bf57c/src/sentry/api/base.py#L440

Reply:
> # Only enforced in dev environment

is the comment that i'm referring to.

the line you linked says

> self.response = self.finalize_response(request, response, *args, **kwargs) 

sorry for the confusion

Reply:
Correction, I accidentally copy-pasted the function again